tests/test_descriptors.py

```diff
@@ -257,10 +257,10 @@ def test___setitem__new(self):
 
     def test___setitem__unicode(self):
         assert self._get_udf_value(self.dict1, 'test') == 'stuff'
-        self.dict1.__setitem__('test', u'unicode')
```

Comment:
... and this - hope we'll at some point be able to forget non-unicode python strings. :champagne: